database/youtube/css.json

```diff
@@ -14,5 +14,13 @@
         "category": "youtube",
         "subcategory": "css",
         "language": "english"
+    },
+    {
+        "name": "DesignCourse",
+        "description": "This channel is great for learning css and frontend. It will give you paid course for css.",
```

Comment:
```suggestion
        "description": "This channel is great for learning css and frontend.",
```

Reply:
I have to remove red line description CBID2 ?